### DISCUSSION TITLE
Eraser to erase specific drawing

### DISCUSSION THREAD

**Discussion by rampatra** [Softal-io/presentify-community#7] [Ideas]
Right now there's an Erase button that clears the entire screen. We do have undo/redo functionality to alter the previous annotation but an eraser (or select and delete) would be beneficial.

**Comment by mikeafish** [comment]
I use a Wacom Intuos Pro - if I could erase using the eraser on the back of the pen that would be even better

**Reply by rampatra** [reply]
This is interesting. I will keep this mind when implementing this.

**Comment by DarkSwoop** [comment]
an eraser would be great!

**Comment by rampatra** [comment]
This is added to version 5.0.0 and above. You can press and hold the Command key and select whichever drawing(s) you want to delete or move.
